src/plone/restapi/services/site/get.py

```diff
@@ -38,11 +42,31 @@ def __call__(self, expand=False):
                 "plone.site_logo": site_settings.site_logo and getSiteLogo() or None,
                 "plone.robots_txt": site_settings.robots_txt,
                 "plone.allowed_sizes": image_settings.allowed_sizes,
+                "plone.timezone": self.plone_timezone(),
```

Comment:
When adding the other settings (site_logo, robots.txt, etc) we kept the registry key name in the output JSON. I would do the same here and use `plone.portal_timezone` instead of `plone.timezone`

Reply:
changed as required